MapboxNavigation/Constants.swift

```diff
@@ -15,6 +15,8 @@ public let RouteControllerProgressDidChange = Notification.Name(MBRouteControlle
 public let RouteControllerAlertLevelDidChange = Notification.Name(MBRouteControllerAlertLevelDidChange)
 public let RouteControllerShouldReroute = Notification.Name(MBRouteControllerShouldReroute)
 
+public let RouteControllerDidReroute = Notification.Name(MBRouteControllerDidReroute)
```

Comment:
Yeah since the RouteController calls RouteControllerShouldReroute notification and RouteViewController handles it by making another call to get directions, so this notification is used in the success block there instead of the RouteController, so placed the notification key constant there just to follow how it was being done for the other notifications